segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -118,7 +118,9 @@ public void start() throws Exception {
                                                       this.serviceConfig.getListeningPort(), service, tableStoreService,
                                                       autoScaleMonitor.getStatsRecorder(), autoScaleMonitor.getTableSegmentStatsRecorder(),
                                                       tokenVerifier, this.serviceConfig.getCertFile(), this.serviceConfig.getKeyFile(),
-                                                      this.serviceConfig.isReplyWithStackTraceOnError(), serviceBuilder.getLowPriorityExecutor());
+                                                      this.serviceConfig.isReplyWithStackTraceOnError(),
```

Comment:
Any chance we could do something to this constructor? It's getting out of hands. Could we add a `@Builder` tag to it?

If we do add a `@Builder`, then we could potentially get rid of the other 1-2 constructors next to it and supply defaults via the builder object.

Not really related to this PR, but every time I had to modify this area of the code I found myself having to modify a good number of other files too.